docs/trace/getting-started/README.md

```diff
@@ -54,7 +54,7 @@ The `ActivitySource` instance is used to start an `Activity` which represents an
 [OpenTelemetry
 Span](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#span).
 An OpenTelemetry
-[TracerProvider](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#tracerprovider)
+[TracerProvider](#tracerprovider)
```

Comment:
point to spec was not particularly useful for end users... so added a section below to explain TracerProvider, and linking directly to where we explain how to customize it.